packages/components/src/components/PipelineRun/PipelineRun.js

```diff
@@ -27,6 +27,7 @@ import {
 } from '@tektoncd/dashboard-utils';
 
 import { Log, RunHeader, StepDetails, TaskTree } from '..';
+import { updateUnexecutedSteps } from '../../../../../src/utils';
```

Comment:
code in packages/* cannot consume code directly from the core dashboard, it must be available in a published npm package (in this case it may make sense to move it to packages/utils) otherwise it will not work for 3rd parties consuming the @tektoncd/dashboard-components package.

Reply:
@AlanGreene So, what would be the best way for me to make this function available? It's used in the following files:
- `src/containers/TaskRun/TaskRun.js`
- `packages/components/src/components/Task/Task.js`
- `packages/components/src/components/PipelineRun/PipelineRun.js`

If I move the function declaration to `packages/utils`, then can I import it from `src/containers/TaskRun/TaskRun.js`?

Reply:
yes that'll work 👍 
you'll just need to import it from `@tektoncd/dashboard-utils` rather than by local path